src/App.js

```diff
@@ -26,16 +34,43 @@ class SimpleBox extends Component {
           this.setState(() => {
             return { clicked: !this.state.clicked };
           })}
+        color={this.props.color}
       />
     );
   }
 }
 
+class ColumnOfBoxes extends Component {
+  render() {
+    return (
+      <div className="columnOfBoxes">
+        <SimpleBox color={this.props.color} />
+        <SimpleBox color={this.props.color} />
+        <SimpleBox color={this.props.color} />
+        <SimpleBox color={this.props.color} />
+      </div>
+    );
+  }
+}
+
+class Box extends Component {
```

Comment:
Extract this to a separate file and follow the proper structure of the project.
Read this article https://hackernoon.com/the-100-correct-way-to-structure-a-react-app-or-why-theres-no-such-thing-3ede534ef1ed